web/client/components/TOC/Toolbar.jsx

```diff
@@ -327,6 +330,15 @@ class Toolbar extends React.Component {
                             </Button>
                         </OverlayTrigger>
                         : null}
+                    {this.props.activateTool.activateSwipeOnLayer && (status === 'LAYER') &&
```

Comment:
I think that also this can be externalized, and this is a good moment to provide this functionality. I'll prepare a patch that helps to render buttons as items in TOC toolbar, but we can do it in a separate issue.